golem_cli/src/setup.rs

```diff
@@ -87,14 +98,18 @@ pub async fn setup(run_config: &RunConfig, force: bool) -> Result<i32> {
             run_config.account.network, account_msg
         );
 
-        while let Some(account) = promptly::prompt_opt::<String, _>(&message)? {
-            match account.parse::<NodeId>() {
-                Err(e) => eprintln!("Invalid ethereum address, is should be 20-byte hex (example 0xB1974E1F44EAD2d22bB995167A709b89Fc466B6c): {}", e),
-                Ok(account) => {
-                    config.account = Some(account);
-                    break;
+        if interactive {
+            while let Some(account) = promptly::prompt_opt::<String, _>(&message)? {
+                match account.parse::<NodeId>() {
+                    Err(e) => eprintln!("Invalid ethereum address, is should be 20-byte hex (example 0xB1974E1F44EAD2d22bB995167A709b89Fc466B6c): {}", e),
```

Comment:
Is the inconsistent uppercasing intended?

Reply:
This is ethereum address control sum.